### PR TITLE
DDBTEAM-286: Re-enabeled soap endpoint

### PR DESCRIPTION
**Pull request by turegjorup**
SOAP endpoint re-added

### CODE REVIEW

File: src/Controller/SoapController.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use Symfony\Component\Routing\Annotation\Route;
+
+class SoapController extends AbstractController
+{
+    /**
+     * @Route("/2.11/", name="soap_trailing_slash")
+     */
+    public function soap(Request $request, MoreInfoService $moreInfoService, LoggerInterface $statsLogger, $projectDir)
+    {
+        $soapServer = new \SoapServer($projectDir.'/src/Service/MoreInfoService/Schemas/moreInfoService.wsdl', ['cache_wsdl' => WSDL_CACHE_NONE]);
```
Comment (cableman) [inline src/Controller/SoapController.php]: Should be spaces between string concatenations.

File: src/Service/MoreInfoService/Exception/MoreInfoException.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: turegjorup
+ * Date: 2018-11-28
+ * Time: 19:56.
```
Comment (cableman) [inline src/Service/MoreInfoService/Exception/MoreInfoException.php]: This is not at @file doc-block :-)

File: src/Service/MoreInfoService/MoreInfoService.php [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+ * moreInfoService class.
+ *
+ * @author wsdl2php
+ */
+class MoreInfoService extends SoapClient
+{
+    /**
+     * Namespace for service calls.
+     *
+     * @TODO update namespace uri when we know the cover service URL
```
Comment (cableman) [inline src/Service/MoreInfoService/MoreInfoService.php]: TODO should be removed as namespace have been set below.

**Review by cableman — CHANGES_REQUESTED** [review]

File: src/Controller/SoapController.php (hunk 1 above)
Comment (turegjorup) [reply src/Controller/SoapController.php]: Not according to our CS FIX config....

**Comment by turegjorup** [comment]
@cableman Updated PR

File: src/Controller/SoapController.php (hunk 1 above)
Comment (cableman) [reply src/Controller/SoapController.php]: okay, just looks so wrong in my eyes :-|

**Review by cableman — APPROVED** [review]
There are some missing doc-blocks for parameter descriptions, but as the hopefully will not exists for to long. I don't think we should use too much time on function level documentation right now for this part of the code base.

**Review by cableman — APPROVED** [review]

**Pull request merged**